plugin/federation/federation.gotpl

```diff
@@ -160,14 +160,13 @@ func (ec *executionContext) __resolve_entities(ctx context.Context, representati
 						{{- end -}}
 						list[idx[i]] = entity
 					}
+					return nil
 				{{ end }}
-			{{- end }}
```

Comment:
I forget what this does again?

Reply:
For removing  new lines.  Formatting of the generated code.